py/surveyqa/nightly.py

```diff
@@ -496,6 +517,25 @@ def makeplots(night, exposures, tiles, outdir):
     </head>
     """
 
+# TODO: this the the navigation link template without hrefs
+#       to be updated later, but that conceptually conflicts with
+#       and earlier PR that was handling the prev/next links on
+#       the first/last nights
+#
+#     template += """
+#     <body>
+#         <ul>
+#           <li style="float:left"><a>DESI Survey QA Night {}</a></li>
+#           <li><a id="last">Last</a></li>
+#           <li><a id="next">Next</a></li>
+#           <li><a id="prev">Previous</a></li>
+#           <li><a id="first">First</a></li>
+#           <li><a href={}>Summary Page</a></li>
+#         </ul>
+#     """.format(night, summary_str)
+
+# TODO: this is the link code from the earlier PR
+
```

Comment:
I resolved merge conflicts, but there is a "how should we do it" conceptual conflict with the previously merged PR #63 .  I left both in here and let's chat as a group on Monday for how to handle the navigation links for the first and last nights in light of this PR method.